prs_per_repo.rb

```diff
@@ -70,73 +77,129 @@ def execute_query_in_repo_or_org(repo, query)
   end
 
-  def closed_merged_during_sprint(repo)
-    execute_query_in_repo_or_org(repo, "is:merged #{closed_during_sprint_search_query}")
+  def issues_closed_during_sprint(repo)
+    execute_query_in_repo_or_org(repo, "type:issue #{closed_during_sprint_search_query}")
   end
 
-  def closed_unmerged_during_sprint(repo)
-    execute_query_in_repo_or_org(repo, "is:unmerged #{closed_during_sprint_search_query}")
+  def prs_merged_during_sprint(repo)
+    execute_query_in_repo_or_org(repo, "type:pr is:merged #{closed_during_sprint_search_query}")
   end
 
-  def created_during_sprint(repo)
-    execute_query_in_repo_or_org(repo, "type:pr created:#{@sprint.range_iso8601}")
+  def prs_closed_during_sprint(repo)
+    execute_query_in_repo_or_org(repo, "type:pr is:unmerged #{closed_during_sprint_search_query}")
+  end
+
+  def created_during_sprint_query
+    "created:#{@sprint.range_iso8601}"
+  end
+
+  def prs_created_during_sprint(repo)
+    execute_query_in_repo_or_org(repo, "type:pr #{created_during_sprint_query}")
+  end
+
+  def issues_created_during_sprint(repo)
+    execute_query_in_repo_or_org(repo, "type:issue #{created_during_sprint_query}")
   end
 
   LABELS  = ["bug", "enhancement", "developer", "documentation", "performance", "refactoring", "technical debt", "test"]
 
-  def process_repo(repo)
-    puts "Analyzing Repo: #{repo}"
+  def process_prs(repo)
+    result = {}
+    counts = {}
 
-    stats = {}
-    stats['repo_slug'] = repo
-    stats['repo_url']  = "http://github.com/#{repo}"
-    stats['prs']    = {}
-    stats['counts'] = {}
+    prs_created = prs_created_during_sprint(repo)
+    result['created'] = prs_created.collect(&:number).sort
+    counts['created'] = prs_created.length
+
+    prs_still_open = prs_remaining_open(repo) + prs_closed_or_merged_after_sprint(repo)
+    result['still_open'] = prs_still_open.collect(&:number).sort
+    counts['still_open'] = prs_still_open.length
 
-    opened                    = created_during_sprint(repo)
-    stats['prs']['opened']    = opened.collect(&:number).sort
-    stats['counts']['opened'] = opened.length
+    prs_closed = prs_closed_during_sprint(repo)
+    result['closed'] = prs_closed.collect(&:number).sort
+    counts['closed'] = prs_closed.length
 
-    still_open = remaining_open(repo) + closed_after_sprint(repo)
-    stats['prs']['still_open']    = still_open.collect(&:number).sort
-    stats['counts']['still_open'] = still_open.length
+    prs_merged   = prs_merged_during_sprint(repo)
+    result['merged'] = prs_merged.collect(&:number).sort
+    counts['merged'] = prs_merged.length
 
-    closed_merged   = closed_merged_during_sprint(repo)
-    closed_unmerged = closed_unmerged_during_sprint(repo)
-    labels_array    = []
-    closed_merged.each do |pr|
+    labels_array = []
+    prs_merged.each do |pr|
       pr.labels.each { |label| labels_array << label.name }
     end
+    result['merged_labels'] = labels_array.element_counts.sort.to_h
```

Comment:
Minor but this can all be one-lined using flat_map to 

```ruby
result['merged_labels']  = prs_merged.flat_map { |pr| pr.labels.collect(&:name) }.element_counts.sort.to_h
```

Reply:
Implemented and pushed.